torchaudio/csrc/build_rir.cpp

```diff
@@ -0,0 +1,134 @@
+#include <math.h>
+#include <torch/script.h>
+#include <torch/torch.h>
+using namespace torch::indexing;
+
+namespace torchaudio {
+namespace rir {
+
+template <typename scalar_t>
+void build_rir_impl(
```

Comment:
Can you add docstring / reference / description?
We do not have a way to build C++ doc yet, but we need make notes for future maintainers.